openedx/core/djangoapps/user_api/management/commands/retire_user.py

```diff
@@ -62,9 +63,14 @@ def check_user_exist(self, user_model, userfile, user_name, useremail):
         users = []
         user = user_name
         email = useremail
-        if userfile and (not user and not email):
+        bulk_user_retirement_config = BulkUserRetirementConfig.current()
+        if userfile and (not user and not email) or not userfile and not user and not email:
```

Comment:
Can we make this more pythonic / clear?

Reply:
or `no_users_identified_via_cli_flags`, say